Jenkinsfile

```diff
@@ -61,6 +61,7 @@ for(j = 0; j < jdks.size(); j++) {
                 stage("${buildType} Publishing") {
                     if (runTests) {
                         junit healthScaleFactor: 20.0, testResults: '*/target/surefire-reports/*.xml'
+                        archiveArtifacts allowEmptyArchive: true, artifacts: '**/target/surefire-reports/*.dumpstream'
```

Comment:
→ https://github.com/jenkinsci/remoting/pull/315

Reply:
interesting, wasn''t aware of this feature. NIT then reading https://maven.apache.org/surefire/maven-surefire-plugin/faq.html#dumpfiles don't we actually want *.dump* in case?

Reply:
I am not sure what `*.dump` is exactly, but anyway an actual `*.dumpstream` file led me to a source code issue. Can always improve as needed.